bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/builders/AutomationChannelBuilder.java

```diff
@@ -134,6 +134,11 @@ public AutomationChannelBuilder withScheduledEvents(List<ScheduledEvent> schedul
      * @return the {@link Channel} list
      */
     public List<Channel> build() {
+        if (scheduledEvents == null || (scenes == null && sceneCollections == null)) {
+            return this.getChannelList(0);
```

Comment:
Probably don't need the `this.` prefix on `this.getChannelList()` ?? , and ditto three lines further down.

Reply:
Fixed.